cocos/core/renderer/scene/model.ts

```diff
@@ -286,147 +286,147 @@ export class Model {
     }
 
     /**
-     * @en model type
+     * @en The type of the model
      * @zh 模型类型
      */
     public type = ModelType.DEFAULT;
 
     /**
-     * @en model's render scene
+     * @en The render scene of the model
```

Comment:
I think the render scene is not part of the model.
Maybe something like "The render scene where the model lives."